### DISCUSSION TITLE
How to write converted rules in a file using command line?

### DISCUSSION THREAD

**Discussion by Iqi-Malick** [SigmaHQ/sigma#3229] [Q&A] [answered]
Currently I am converting sigma rules to elastalert using sigmac. I executed following command 
`cd /sigma/tools `
`./sigmac -I -t elastalert  -r rules/network/zeek/ -c ecs-zeek-elastic-beats-implementation`
After this I am able to see converted rule on the terminal only. I want to store these converted rules in separate files. Ca anyone tell how can I achieve this?

**Comment by frack113** [comment, marked as answer]
` python sigmac -I -t elastalert -r ../rules/network/zeek/ -c ecs-zeek-elastic-beats-implementation -o d:/zeek/ -e yml`

Will genere in the folder `d:/zeek/`  files with extention `yml`

**Reply by Iqi-Malick** [reply]
Thank you. It worked.
